mlkem/src/fips202/fips202x4.c

```diff
@@ -78,7 +79,8 @@ static void mlk_keccak_squeezeblocks_x4(uint8_t *out0, uint8_t *out1,
                                         size_t nblocks, uint64_t *s, uint32_t r)
 __contract__(
     requires(r <= sizeof(uint64_t) * MLK_KECCAK_LANES)
-    requires(nblocks <= 8 /* somewhat arbitrary bound */)
```

Comment:
Why was this changed?

Reply:
If the caller is only restricted to MLK_MAX_BUFFER_SIZE bytes, that's a lot more than 8 blocks I think. Proof fails otherwise.

Reply:
Ack